From ec63dfcaeeef84fdab01319fb925198309969c60 Mon Sep 17 00:00:00 2001
From: Anish <anish.choudhary@siriusemb.com>
Date: Tue, 29 May 2018 18:49:40 +0530
Subject: [PATCH] "new commit"

---
 cronie/crontab | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/cronie/crontab b/cronie/crontab
index 22c4feb..2a25c15 100644
--- a/cronie/crontab
+++ b/cronie/crontab
@@ -12,3 +12,14 @@ PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
 # 30  7     * * *    root        cd / && run-parts /etc/cron.daily
 # 42  7     * * 7    root        cd / && run-parts /etc/cron.weekly
 # 55  7     1 * *    root        cd / && run-parts /etc/cron.monthly
+
+# Example of job definition:
+# .---------------- minute (0 - 59)
+# |  .------------- hour (0 - 23)
+# |  |  .---------- day of month (1 - 31)
+# |  |  |  .------- month (1 - 12) OR jan,feb,mar,apr ...
+# |  |  |  |  .---- day of week (0 - 6) (Sunday=0 or 7) OR sun,mon,tue,wed,thu,fri,sat
+# |  |  |  |  |
+# *  *  *  *  *     user-name command to be executed
+
+*/1  *  *  *  *   root    /etc/logrotate.d/xl4-logrotate
-- 
2.7.4

